extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -48,7 +59,10 @@ export class AnchorAdStrategy {
       return Promise.resolve(false);
     }
     if (this.hasExistingStickyAd_()) {
-      user().warn(TAG, 'exists <amp-sticky-ad>');
+      user().warn(
+        TAG,
+        'Unable to create an anchor ad because already exists <amp-sticky-ad>'
```

Comment:
I'm wondering if this should more specifically address the auto ads creation case (requiring that you disambiguate if you created the anchor just for crawl vs pub had enabled)... up to you.  You could indicate what network has an existing item and which is attempting to add to make this more actionable.

Reply:
Created separate warning for fill anchor ad and no fill anchor ad. Thanks!!